src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1307,8 +1307,9 @@ class StartSessionHandler : public security_manager::SecurityManagerListener {
 }  // namespace
 #endif  // ENABLE_SECURITY
 
-// DEPRECATED
-RESULT_CODE ProtocolHandlerImpl::HandleControlMessageStartSession(
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
@jacobkeeler please describe cases when you use ignoring -Wdeprecated-declarations diagnostic